### PR TITLE
Events/Tickets Menus > Tickets menu

### PR DESCRIPTION
**Pull request by juanfra**
### 🎫 Ticket

[ET-1335] 

### 🗒️ Description

- Implementation of top of https://github.com/the-events-calendar/tribe-common/pull/1691

### 🎥 Artifacts 

- Coming soon.

### ✔️ Checklist
- [ ] I've included a changelog entry in the readme.txt file. 
- [ ] My code is tested. 
- [ ] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1335]: https://theeventscalendar.atlassian.net/browse/ET-1335?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tribe/Admin/Ticket_Settings.php [hunk 1]
```diff
@@ -36,4 +245,134 @@ protected function get_settings_array() {
...
+		new Tribe__Settings_Tab( 'network', esc_html__( 'Network', 'event-tickets' ), $networkTab );
+	}
+
+	/**
+	 * Add the Event Tickets admin footer text.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $footer_text The admin footer text.
+	 * @param string $footer_text The admin footer text, maybe modified.
```
Comment (redscar) [inline src/Tribe/Admin/Ticket_Settings.php:300]: Document @return

File: src/admin-views/settings/tickets-commerce/paypal/connect/active/actions/refresh-connection.php [hunk 2]
```diff
@@ -23,7 +23,7 @@
 	return;
 }
 
-$resync_connection_url = Tribe__Settings::instance()->get_url( [
+$resync_connection_url = Tribe__Tickets__Main::instance()->settings()->get_url( [
```
Comment (borkweb) [inline src/admin-views/settings/tickets-commerce/paypal/connect/active/actions/refresh-connection.php:26]:
```suggestion
$resync_connection_url = tribe( 'tickets.main' )->settings()->get_url( [
```

File: src/admin-views/settings/tickets-commerce/paypal/connect/active/actions/refresh-user-info.php [hunk 3]
```diff
@@ -25,7 +25,7 @@
 	return;
 }
 
-$url = Tribe__Settings::instance()->get_url( [
+$url = Tribe__Tickets__Main::instance()->settings()->get_url( [
```
Comment (borkweb) [inline src/admin-views/settings/tickets-commerce/paypal/connect/active/actions/refresh-user-info.php:28]:
```suggestion
$url = tribe( 'tickets.main' )->settings()->get_url( [
```

**Review by borkweb — CHANGES_REQUESTED** [review]
This is looking pretty sweet!

**Review by borkweb — APPROVED** [review]

**Review by borkweb — CHANGES_REQUESTED** [review]
Ah, I need to retract my approval - I realized something that needs a bit of a refactor while looking at the TEC implementation. I'm working on a PR for that and then it can be replicated on this PR as well.

**Comment by borkweb** [comment]
Here's a re-working of the class structure for TEC. We'll need to replicate that for Event Tickets as well: https://github.com/the-events-calendar/the-events-calendar/pull/3813

**Comment by juanfra** [comment]
@borkweb excellent, thanks! [Here's a commit with those changes for ET](https://github.com/the-events-calendar/event-tickets/pull/2228/commits/75ea73ca55c365e2fdd61a83d4c61fd8636fb495).

I went ahead and renamed the class to `Tickets_Settings` ([related to this comment](https://github.com/the-events-calendar/the-events-calendar/pull/3813#discussion_r849217178)), as it makes more sense and it's more in-line with our naming conventions.

File: src/Tribe/Admin/Tickets_Settings.php [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
 
 use Tribe\Admin\Troubleshooting as Troubleshooting;
+use Tribe__Settings;
+use Tribe__Settings_Tab;
+
 /**
  * Manages the admin settings UI in relation to ticket configuration.
  */
-class Tribe__Tickets__Admin__Ticket_Settings {
+class Tickets_Settings {
```
Comment (borkweb) [inline src/Tribe/Admin/Tickets_Settings.php:11]: This should be `Ticket_Settings` - See my comment [here](https://github.com/the-events-calendar/the-events-calendar/pull/3813/files#r849440547) for a bit more info.

**Review by borkweb — CHANGES_REQUESTED** [review]

File: src/Tribe/Admin/Tickets_Settings.php (hunk 4 above)
Comment (borkweb) [reply src/Tribe/Admin/Tickets_Settings.php:11]: Revising based on our conversation in Slack and the resulting update to TEC - change to `src/Tribe/Admin/Settings.php`

**Review by borkweb — APPROVED** [review]

**Pull request merged**